drivers/console/bm_serial.c

```diff
@@ -458,73 +344,48 @@ static void bm_serial_tx_thread(void)
 	uint8_t n;
 	uint16_t encoded_val;
 
+	/* Create a buf to pass to DMA TX (account for preamble)*/
+	uint8_t tx_enc_buf[2*(MAX_BM_FRAME_SIZE) + BM_PREAMBLE_LEN];
+	uint16_t tx_buf_ctr = 0;
+
 	while (1) 
 	{
 		k_msgq_get(&tx_queue, &msg, K_FOREVER);
 
 		frame_addr = msg.frame_addr;
 		frame_length = msg.frame_length;
```

Comment:
Should we check that the message length is less than the maximum somewhere? Don't want to overflow `tx_enc_buf`!